content/building-blocks/automate-and-execute-your-work/automate-your-workflow/ghactions-workflow.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Build a workflow on a GitHub hosted runner" 
+description: "Learn a practical example of a GitHub actions workflow for code formatting and testing"
+keywords: "GitHub Actions, Data Science, Project, Project Structure, Reproducibility, Automation, Testing, Formatting, Workflows, Runners, Jobs, Events, Testing, Formatting"
+weight: 2
+author: "Fernando Iscar"
+authorlink: "https://www.linkedin.com/in/fernando-iscar/"
+draft: false
+date: 2023-10-14T22:01:14+05:30 
+aliases: 
+  - /github_actions/workflow
+---
+
+## Overview
+
+This building block delves deeper into GitHub Actions by providing a hands-on example. We’ll craft a workflow specifically for code formatting and testing. After that, you'll discover how to integrate and track it into your own repository.
+
+{{% tip %}}
+If you are new to GitHub Actions, we suggest you review [this building block](https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/automate-your-workflow/intro_ghactions/) first to get familiar with the foundational concepts.
+
+{{% /tip %}}
+
+Hence, by the end of this block, you will:
+
+- Know how to set up a GitHub Actions workflow for code formatting and testing.
+- Learn how to monitor your workflow from the GitHub interface.
+
+## Building a GitHub Actions Workflow
+
+In this section, we're going to build upon the `.yml` file structure introduced in the previous [guide](https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/automate-your-workflow/intro_ghactions/). 
+
+To illustrate a useful workflow example, our goal would be to integrate code formatting (using Super Linter) and code testing.
+
+In case you're interested, the example was retrieved from the following GitHub repository:
+
+{{% cta-primary-center "Click to go to the repository example" "https://github.com/snpe-reputation-systems/snpe-reputation-systems/blob/master/.github/workflows/formatting-testing.yml" %}}
+
+### Code Formatting with Super Linter
+
+[Super Linter](https://github.com/marketplace/actions/super-linter) is a simple combination of various linters to help validate source code. It works automatically whenever your workflow is activated by an event, like pushing new code. 
```

Comment:
Again, this is good, but for further clarity I would rephrase to "The Super Linter action" at the beginning of the sentence for further clarity.